modules/retail/pages/retail-install-uyuni.adoc

```diff
@@ -57,9 +57,18 @@ zypper in bind-formula \
 	  vsftpd-formula
 
 ----
-. Reboot the client.
+. Reboot the {productname} {smr} Server.
 
-// yast2 susemanager_setup
 
-When the installation is complete, you can continue with {productname} setup.
-For more information, see xref:retail:retail-uyuni-server-setup.adoc[].
+.Procedure: Running the Setup Script on the {productname} {smr} Server
+. On the {productname} {smr} Server, at the command prompt, as root, run the setup script:
++
+----
+yast2 susemanager_setup
+----
+. Follow the prompts to set up your account.
+  Take note of the passwords you set, you will need them later on.
+
+
+Continue with general {productname} configuration and channel synchronization.
+
```

Comment:
I do not think this is necessary. I would leave the original text and instead fix the xref to point to https://github.com/uyuni-project/uyuni-docs/blob/master/modules/installation/pages/uyuni-server-setup.adoc

Reply:
ok, reverted now and xref provided.